packages/kotti-ui/source/kotti-user-menu/KtUserMenu.vue

```diff
@@ -55,32 +55,38 @@ import { mixin as clickaway } from 'vue-clickaway'
 
 import { KtAvatar } from '../kotti-avatar'
 import { IS_NAVBAR_NARROW } from '../kotti-navbar/constants'
+import { propValidator } from '../props'
 import { Kotti } from '../types'
 
 import { KottiUserMenu } from './types'
 
-const linkIsValid = (link: KottiUserMenu.SectionLink) => Boolean(link.title)
```

Comment:
who wrote those validator expressions? ok, apparently you.

you assumed the type without pre-checking if it's not `unknown` - so for example by having another `Object.prototype.call`